src/integrations/__tests__/legacy.spec.ts

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import removeLegacySymlinks from '@/integrations/legacy';
+
+const resourcesDir = path.join('resources', os.platform(), 'bin');
+let testDir: string;
+let someOtherDir: string;
+let legacyIntegrationDir: string;
+
+beforeEach(async() => {
+  testDir = await fs.promises.mkdtemp(path.join(os.tmpdir(), 'rdtest-'));
+  someOtherDir = path.join(testDir, 'someOtherDir');
+  await fs.promises.mkdir(someOtherDir);
+  legacyIntegrationDir = path.join(testDir, 'legacyIntegrationDir');
+  await fs.promises.mkdir(legacyIntegrationDir);
+});
+
+afterEach(async() => {
+  // It is best to be careful around rm's; we don't want to remove important things.
```

Comment:
This comment doesn't look like it's pulling its weight. You created a temp dir at line 12 and are deleting it here. If something else gets deleted the problem isn't with the RD code.

Reply:
This creating and removal of a temporary directory is copied from the `PATH` management PR. I originally had just a `fs.promises.rm` in `afterEach`, but Mark made [this comment](https://github.com/rancher-sandbox/rancher-desktop/pull/1784#discussion_r824244146) so I changed it to be more careful. I'm not picky as to how we do this, but we should probably make them the same.